### ISSUE TITLE
wrong service blueprint in ng completion generate file

### ISSUE THREAD

**Issue by szaccaria** [angular/angular-cli#5640]
### Bug Report or Feature Request (mark with an `x`)
```
- [ x] bug report -> please search issues before submitting
- [ ] feature request
```

### Versions.

@angular/cli: 1.0.0
node: 6.9.5
os: darwin x64
### Repro steps.

when generate the .bashrc file with 

`ng completion -b >> ~/.bashrc`

it generate the file with a little mistake
In the `case ${pword} in` segment of the file there is a problem with the `opts` list of the blueprint

`g|generate) opts="class component directive enum guard interface module pipe service--dry-run --verbose -d -v" ;;`

the `service` and `--dry-run` are close and present to the completion a wrong blueprint

### The log given by the failure.

```
stefano-3:~ stefano$ ng g 
--verbose         -v                component         enum              interface         pipe              
-d                class             directive         guard             module            service--dry-run  
```
### Desired functionality.

Can add the instruction for install on mac? I need to do

`ng completion -b >> ~/.bash_profile`

mac read .bash_profile no the .bashrc

```
# Installation: ng completion -b >> ~/.bashrc
#           or  ng completion -z >> ~/.zshrc
```

thanks

**Comment by filipesilva** [comment]
Thanks for the report! Should be fixed by https://github.com/angular/angular-cli/pull/5612

**Comment by sumitarora** [comment]
Closing as above is merged and released in 1.0.1 https://github.com/angular/angular-cli/blob/master/CHANGELOG.md#101-2017-04-25
